### DISCUSSION TITLE
PYGAME, my enemies are not shooting

### DISCUSSION THREAD

**Discussion by andyargg** [community/community#73249] [Programming Help]
### Body

Hi, I'm working on a space shooter game in Pygame. This is my first game, and I'm having trouble getting the enemies to shoot. It seems impossible because, when I print the enemy bullets, they are at unexpected coordinates and i cannot see them

You can check the repository here: https://github.com/andyargg/PYGAME_2-master.git

I've been trying to fix this for four days now and could use some help. Thank you

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by faridnec** [comment]
hey i could not find the repo it gives 404 error

**Reply by andyargg** [reply]
I fixed it xd
